gae_discourse_client/client.py

```diff
@@ -0,0 +1,93 @@
+"""Gateway for accessing the Discourse API (for forums)"""
+
+import json
+import re
+from urllib import urlencode
+
+from google.appengine.api import urlfetch
+from google.appengine.ext import ndb
+
+import categories as categories_module
+import groups as groups_module
+import users as users_module
+
+
+class DiscourseAPIClient(object):
```

Comment:
I'm not sure how. If our requests to Discourse are broken, all the other tests will fail anyway, and I'm not aware of any generic HTTP requests (`GET`s or `POST`s or `DELETE`s) that I could make. Did you have any specific tests in mind that you wanted to see?